_targets.R

```diff
@@ -1,9 +1,11 @@
 library(targets)
 
 options(tidyverse.quiet = TRUE)
-tar_option_set(packages = c("tidyverse", "lubridate","rmarkdown","knitr","leaflet","sf")) 
+tar_option_set(packages = c("tidyverse", "lubridate","rmarkdown","dataRetrieval","knitr","leaflet","sf")) 
 
-source("./2_process/src/2_process.R")
+source("1_fetch.R")
+source("2_process.R")
+source("3_visualize.R")
```

Comment:
I'm not 100% sure, but I think USGS practice is to place these in their respective src folders instead of the main directory. I kind of like seeing them in the main repo for quick access, but it's probably not as clean once there are many steps in the pipeline.

Reply:
I think these are needed though so that the _targets.R file knows where to access the lists of targets for the individual phases.

Reply:
Yes, they're definitely needed! I'm just suggesting to move them to the src directories

Reply:
Ah, I see! I first thought you were suggesting moving these calls to source as opposed to just changing the path of `1_fetch.R`. I kind of like keeping the phase targets files in the main directory to separate these lists of targets from the more specific functions in the respective src folders, but I know what you mean about the main directory getting cluttered.

After taking a quick look back at the formatting used in the targets training as well as another USGS-R repo (delaware-model-prep), I decided to keep these files in the main project directory for now, but we can revisit as we add other steps to the pipeline.